src/app/(Authentication)/Login/Email/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { router } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import { emailExists } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [email, setEmail] = useState('');
+  const [isEmail, setIsEmail] = useState(false);
+  const [displayError, setDisplayError] = useState(false);
+  const [displayEmail, setDisplayEmail] = useState(false);
+  const [placeholder, setPlaceholder] = useState('Email');
+  const [isFocused, setIsFocused] = useState(false);
+
+  const onClick = () => {
+    setPlaceholder('');
+    setIsFocused(true);
+    setDisplayEmail(true);
+  };
+
+  const offClick = () => {
+    setPlaceholder('Email address');
+    setIsFocused(false);
+  };
+
+  function removeEmail() {
+    if (email.trim() === '') {
+      setDisplayEmail(false);
+    }
+    if (email.trim() !== '') {
+      setIsEmail(true);
+    } else {
+      setIsEmail(false);
+    }
+    setIsEmail(!isEmail);
+  }
+
+  async function emailFind() {
+    const isEmail = await emailExists(email);
+    if (!isEmail) {
+      setDisplayError(true);
+    } else {
+      setDisplayError(false);
+      router.push({ pathname: 'Login/Password', params: { email } });
+      setEmail('');
+      setDisplayEmail(false);
+    }
+  }
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity style={styles.backButton} onPress={() => router.back()}>
+        <Text style={styles.backText}>Back</Text>
+      </TouchableOpacity>
+      <Text style={styles.instructionText}>
+        Please enter your email address.
+      </Text>
+      <Text style={styles.emailText}>
```

Comment:
We can essentially make a `AuthInput` component that takes in the necessary state variables and other props that does the same thing as this but more generally so it can be used across different screens.